### DISCUSSION TITLE
Mobile Menu Doesn't Close

### DISCUSSION THREAD

**Discussion by steynkalala** [vercel/next.js#28835] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.15.3

### What browser are you using?

Chrome, Safari, Firefox

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

The mobile menu does not close when the Link to the page is clicked. I've tried numerous jsx variations. They all have this problem when <Link> tag is used. The page loads in the background but the menu does not close. I was looking around and found a Strapi/Nextjs starter website, it has the same bug, when you click on the contact page the page loads in the background but the menu doesn't close.

### Expected Behavior

The menu closes after the link is clicked.

### To Reproduce

<img width="521" alt="Screen Shot 2021-09-01 at 1 56 50 PM" src="https://user-images.githubusercontent.com/10123821/131834717-81695396-7048-4a69-8047-21777e7ac1db.png">

**Comment by catnose99** [comment]
It seems the issue is not about Next.js.
https://github.com/strapi/strapi-starter-next-corporate is the right repository to report.

**Comment by steynkalala** [comment]
@catnose99 The strapi reference was an example. Also Strapi is the backend of that reference and an easy example to use. Try it yourself, The menu doesn't work This has to be a NextJS bug.
